jest_config/__fixtures__/mnemonic.ts

```diff
@@ -0,0 +1,2 @@
+export const fMnemonicPhrase =
```

Comment:
Can we use something like `test test test test test test test test test test test test test test test test test test test test test test test blade` (yeah that's a valid one) here just to make sure that nobody will ever use this mnemonic?